XPlat/RadialProgress/nuget/Xamarin.RadialProgress.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.RadialProgress</id>
+    <title>Xamarin Radial Progress</title>
+    <version>1.0.2</version>
+    <authors>Xamarin Inc.</authors>
+    <owners>Xamarin Inc.</owners>
```

Comment:
What should the `<authors>` and `<owners>` be for legal or whatnot.. I have just copied mapbox's "Xamarin"